### DISCUSSION TITLE
why these cve templates are put into ignore list?

### DISCUSSION THREAD

**Discussion by miaochiahao** [projectdiscovery/nuclei-templates#699] [Q&A] [answered]
I noticed that some cve templates are put into ignore list, why?
https://github.com/projectdiscovery/nuclei-templates/blob/7b8bacb359832de6c0c3a0c344896e7074055c7e/.nuclei-ignore#L15-L18

**Comment by ehsandeep** [comment, marked as answer]
@miaochiahao those templates do not exist in the project anymore, we have removed them as those templates resulting in false-positive results, and we have a [bug in template update logic ](https://github.com/projectdiscovery/nuclei/issues/368)with nuclei that do not delete removed templates from a local disk of users, so to make sure users not running invalid/removed templates we are using `.nuclei-ignore` list, once we fix the bug, this is not needed.
